src/sagemaker/predictor.py

```diff
@@ -229,6 +229,20 @@ def _row_to_csv(obj):
     return ','.join(obj)
 
 
+class _CsvDeserializer(object):
+    def __init__(self):
+        self.content_type = CONTENT_TYPE_CSV
+
+    def __call__(self, stream, content_type):
+        try:
+            return list(csv.reader(stream.read().decode('utf-8').splitlines()))
```

Comment:
should we make encoding configurable as well here?

Reply:
good point